lib/xMsExampleExtractor.ts

```diff
@@ -221,11 +221,20 @@ export class XMsExampleExtractor {
                     bodyParamValue !== "" ? JSON.parse(bodyParamValue) : "";
                 }
               }
+
+              const parseResponseBody = (body: any) => {
+                try {
+                  return JSON.parse(body);
+                } catch (err) {
+                  return body;
```

Comment:
Can you log error saying the body is mal-formatted? The example generated is not a valid file.

Reply:
It's extract example from recording and write back example to swagger. So the response payload has some non-json format such as xml. 

We don't log error here. Since it's a trial for parsing json.